framework/doc/content/source/interfaces/Coupleable.md

```diff
@@ -25,6 +25,17 @@ The following tables summarize the methods it provides.
 
 ---
 
+| Methods for array scalar field variables | Description |
```

Comment:
```suggestion
| Methods for array field variables | Description |
```
since there s no ambiguity
note that we do have array scalar variables

Reply:
Done.